src/pages/whats-happening/v11-release/index.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: The v11 release
+description:
+  The Carbon v11 release includes great quality of life updates for designers
+  and developers, and with no brand updates, you can update at your convenience.
+---
+
+<PageDescription>
+
+The Carbon v11 release includes great quality of life updates for designers and
+developers, and with no brand updates, you can update at your convenience.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Read more in Medium</AnchorLink>
+<AnchorLink>Release FAQs</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+Carbon v11 focuses on quality of life updates for designers and developers
+including token and prop renaming for ease of use, component API updates for a
+more predictable developer experience, and introducing CSS grid for accurate
+layout building in line with the design language.
+
+With no changes to the IBM Design Language, v11 will not require any
+brand-driven product redesigns and teams are free to update at their
+convenience.
+
+### What’s new
+
+- CSS grid lets you more accurately build layouts in line with the design
+  language
+- Inline theming for light/dark modes
+- Token renaming for better understanding and ease of use
+- Size props introduced for icons to reduce icon package size
+- Reduced compile times with new Sass modules
+
+### What’s not changing
+
+- No changes to the IBM Design Language
+- No need to rebuild your UI
+- Tokens that are changing will continue to work in v11
+- There’s no rush to update if you don’t want to immediately take advantage of
```

Comment:
Feedback from Phil -> Change from “no rush to update” to “when we will stop support for v10"